dockerfiles/alpine/is/Dockerfile

```diff
@@ -106,14 +108,9 @@ RUN \
     && apk add unzip \
     && apk add wget
 
-## set the user and work directory
-USER ${USER_ID}
-WORKDIR ${USER_HOME}
-
```

Comment:
Lets not remove these lines as this section is used to allow copying zip files to docker context